examples/ola-throughput-multi.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ola-throughput-multi.cpp
+ * Send a bunch of frames quickly on multiple universes to load test the server.
+ * Copyright (C) 2022 Perry Naseck (git@perrynaseck.com)
+ * Copyright (C) 2005 Simon Newton (this file based on ola-throughput.cpp)
```

Comment:
We don't bother with more than one copyright line generally (even when heavily inspired by something else...
```suggestion
```

Reply:
Fixed!